### ISSUE TITLE
如何使用label_anything只做分割标注？根据教程标注出来的是目标检测加分割？

### ISSUE THREAD

**Issue by wangqiuji1** [open-mmlab/playground#153]
<img width="500" alt="1693062495884" src="https://github.com/open-mmlab/playground/assets/58511832/c94dd929-0460-40cd-9d94-a8d669ad2edc">

**Comment by dky233** [comment]
只输出多边形标注就可以了，如下：
label-studio-ml start sam --port 8003 --with \
  model_name=mobile_sam  \
  sam_config=vit_t \
  sam_checkpoint_file=./mobile_sam.pt \
  device=cpu 
 out_poly=True

**Comment by JimmyMa99** [comment]
教程做的是实例分割（mask-rcnn），我们推荐使用rle格式输出的coco，也就是脚本输出的rle coco格式

**Comment by hbfast** [comment]
我发现hq-sam好像支持不了，
Unexpected key(s) in state_dict: "mask_decoder.hf_token.weight", "mask_decoder.hf_mlp.layers.0.weight", "mask_decoder.hf_mlp.layers.0.bias", "mask_decoder.hf_mlp.layers.1.weight", "mask_decoder.hf_mlp.layers.1.bias", "mask_decoder.hf_mlp.layers.2.weight", "mask_decoder.hf_mlp.layers.2.bias", "mask_decoder.compress_vit_feat.0.weight", "mask_decoder.compress_vit_feat.0.bias", "mask_decoder.compress_vit_feat.1.weight", "mask_decoder.compress_vit_feat.1.bias", "mask_decoder.compress_vit_feat.3.weight", "mask_decoder.compress_vit_feat.3.bias", "mask_decoder.embedding_encoder.0.weight", "mask_decoder.embedding_encoder.0.bias", "mask_decoder.embedding_encoder.1.weight", "mask_decoder.embedding_encoder.1.bias", "mask_decoder.embedding_encoder.3.weight", "mask_decoder.embedding_encoder.3.bias", "mask_decoder.embedding_maskfeature.0.weight", "mask_decoder.embedding_maskfeature.0.bias", "mask_decoder.embedding_maskfeature.1.weight", "mask_decoder.embedding_maskfeature.1.bias", "mask_decoder.embedding_maskfeature.3.weight", "mask_decoder.embedding_maskfeature.3.bias".

**Comment by JimmyMa99** [comment]
问题已经解决，请看 https://github.com/open-mmlab/playground/pull/164

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
